{
    "type": "bundle",
    "id": "bundle--5f5eb12c-5969-438b-b649-3c0ceb8da727",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--fee3adba-47da-441a-87e9-2192c51cde74",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-03-11T01:14:34.481726Z",
            "modified": "2025-03-11T01:14:34.481726Z",
            "name": "CVE-2022-48610",
            "description": "This issue was addressed through improved state management. This issue is fixed in macOS Ventura 13.1, watchOS 9.2, iOS 16.2 and iPadOS 16.2. An app may be able to access user-sensitive data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-48610"
                }
            ]
        }
    ]
}